### ISSUE TITLE
[1.12.1] Transformers not rotating

### ISSUE THREAD

**Issue by craft90** [TechReborn/TechReborn#1243]
The energy system is giving me a bit of a headache recently. Sure it doesn't seem like things burn up or go boom if they get more power than able to handle. However I do try to keep transformers up just in case. I have noticed some issues tho.

1. You can't rotate them. Which I don't know if it's intended or not.

2. Which sides do you connect too? I assume the big "hole" connects to the higher tier but when I tried that it didn't receive any energy. So I changed it around until I noticed...

3. ... Transformers can't seem to be able to connect to blocks, only cables. E.g. trying to have the HV transformers connecting to the fusion reactor or have a LV transformer connect to a battery box. Putting a cable between seems to have solved the issue. However I have no way of knowing if the energy is going through the right way. This parried with cables generating free energy and things NOT blowing up is giving me a headache. 

Also, the Diesel generator isn't completely compatible with JEI. Going through the fuels in liquid form eventually leads me to a transposer or squeezer which doesn't tell me how to make said fuel. I had to go through the fluid cells to figure out what to do. Btw is oil lakes implemented or not? I don't find it in JEI nor when mining.

**Comment by craft90** [comment]
Another little glitch... so I have both medium producers/users and low producers/users. I have connected them to the same system with transformers separating low tiers from medium tier. However, the power isn't going through the LV-transformer. So the Battery box runs out of power but no power is input. So I removed the transformer and have cables, gold cables, connecting directly to the low tier system only passing through a medium tier battery. Power is going through now and nothing explodes. But I don't think that's intended. What would the use of transformers be otherwise?

EDIT: In latest build power seems to go through. But from small to big which I find illogical. IRL industries do have these hi-voltage cables aka 5 phase which are significantly larger than a regular contact (5 pins instead of 2 + ground). I believe the case is the same in IC2? They act more like batteries which only have the one output.
